src/store/ChartStore.js

```diff
@@ -899,6 +916,35 @@ class ChartStore {
         this.chartClosedOpenThemeChange(!symbolObj.exchange_is_open);
     }
 
+    /**
+     * load the chart with given data
+     *
+     * by this methos, beside of waiting for Feed@fetchInitialData to provide first data
+     * the chart are initiled by give masterData. Chart need a symbol to be able to get
+     * loaded, so if the passed symbol didn't fill, it try to get the symbol from `layout-*`
+     * storage
+     *
+     * @param {string} symbol the symbol used to load the chart
+     * @param {array} masterData array of ticks regards of desire tick
+     */
+    loadChartWithInitalData(symbol, masterData) {
+        if (!masterData) return;
+
+        const layoutData = createObjectFromLocalStorage(`layout-${this.stateStore.chartId}`);
+        if (!layoutData || !layoutData.symbols.length) return;
+        const layout_symbol = layoutData.symbols[0].symbol;
+
+        this.stxx.loadChart(symbol || layout_symbol || '1HZ10V', {
```

Comment:
Why do we need `1HZ10V` here?

Reply:
@zaki-binary chart needs a `symbol` for load and this is in case the user forgot to pass any symbol. we can also replace this with an error.

Reply:
Maybe replacing it with an error is better? So the user is aware.

Reply:
@zaki-binary fixed